public/scripts/app.js

```diff
@@ -121,16 +83,17 @@ $('ul.nav li').click(function(){
         `)
      }
    }
-   function renderAllProjects(data){
```

Comment:
You use `data[0]` everywhere in this function which is 1) not DRY, and 2) not descriptive. 

I changed the parameter name to be `profile` (because that's what `data[0]` is – the first profile), and changed the argument that I passed into this function when I invoke it.